apps/chat-e2e/src/tests/sharedFilesAttachments.test.ts

```diff
@@ -0,0 +1,477 @@
+        await chatBar
+          .getBottomDropdownMenu()
+          .selectMenuOption(MenuOptions.attachments);
+        await attachedAllFiles.waitForState();
+
+        await attachedAllFiles.expandFolder('appdata', {
+          isHttpMethodTriggered: true,
+        });
+        await attachedAllFiles.expandFolder(defaultModel, {
+          isHttpMethodTriggered: true,
+        });
+        await attachedAllFiles.expandFolder('images', {
+          isHttpMethodTriggered: true,
+        });
+        await attachedAllFiles.expandFolder(specialCharsFolder, {
+          isHttpMethodTriggered: true,
+        });
+
+        await attachedAllFiles.getFolderByName('images').hover();
+
+        const firstImageEntity: TreeEntity = { name: Attachment.sunImageName };
+        await attachedFilesAssertion.assertSharedFileArrowIconState(
+          firstImageEntity,
+          'visible',
+        );
+        await attachedFilesAssertion.assertEntityArrowIconColor(
+          firstImageEntity,
+          Colors.controlsBackgroundAccent,
+        );
+
+        const secondImageEntity: TreeEntity = {
+          name: Attachment.cloudImageName,
+        };
+        await attachedFilesAssertion.assertSharedFileArrowIconState(
+          secondImageEntity,
+          'visible',
+        );
+        await attachedFilesAssertion.assertEntityArrowIconColor(
+          secondImageEntity,
+          Colors.controlsBackgroundAccent,
+        );
+
+        const thirdImageEntity: TreeEntity = {
+          name: Attachment.flowerImageName,
+        };
+        await attachedFilesAssertion.assertSharedFileArrowIconState(
+          thirdImageEntity,
+          'visible',
+        );
+        await attachedFilesAssertion.assertEntityArrowIconColor(
+          thirdImageEntity,
+          Colors.controlsBackgroundAccent,
+        );
+
+        //TODO EPMRTC-4135 blocked by the #1076
+        // const fourthImageEntity: TreeEntity = { name: Attachment.heartImageName };
+        // await attachedFilesAssertion.assertSharedFileArrowIconState(fourthImageEntity, 'visible');
+        // await attachedFilesAssertion.assertEntityArrowIconColor(fourthImageEntity, Colors.controlsBackgroundAccent);
+
+        const specialCharsImageEntity: TreeEntity = {
+          name: Attachment.specialSymbolsName,
+        };
+        await attachedFilesAssertion.assertSharedFileArrowIconState(
+          specialCharsImageEntity,
+          'visible',
+        );
+        await attachedFilesAssertion.assertEntityArrowIconColor(
+          specialCharsImageEntity,
+          Colors.controlsBackgroundAccent,
+        );
+        await attachFilesModal.closeButton.click();
+      },
+    );
+
+    await dialSharedWithMeTest.step(
+      'Prepare conversation to share of the user 2',
+      async () => {
+        additionalShareUserConversationData.resetData();
+        const conversationToShare =
+          additionalShareUserConversationData.prepareEmptyConversation(
```

Comment:
Please, avoid references to specific models

Reply:
done